src/modules/leaderboard.js

```diff
@@ -0,0 +1,74 @@
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/PYiYgQE33xPmsyutxO0U/scores';
+
+const listScores = () => document.querySelector('#list-scores');
+
+const checkEmpty = () => {
+  if (listScores().innerHTML === '') {
+    const listItem = document.createElement('li');
+    listItem.textContent = 'Recorded No Scores';
+    listScores().appendChild(listItem);
+  }
+};
+
+const scoresDisplay = (scores) => {
+  scores.forEach((scoreObject) => {
+    const listItem = document.createElement('li');
+    listItem.textContent = `${scoreObject.user}: ${scoreObject.score}`;
+    listScores().appendChild(listItem);
+  });
+};
+
+const getScoreObjects = async () => {
+  const response = await fetch(url);
+  const scores = (await response.json()).result;
+
+  scoresDisplay(scores);
+};
+
+const refresh = async () => {
+  listScores().innerHTML = '';
+  await getScoreObjects();
+  checkEmpty();
+};
+
+const addRefresh = () => {
+  const button = document.querySelector('#refresh');
+  button.addEventListener('click', () => {
+    refresh();
+  });
+};
+
+window.addEventListener('load', () => {
+  refresh();
+});
+
+addRefresh();
+
+const postScore = (scoreObject) => {
+  fetch(url, {
+    method: 'POST',
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+    body: JSON.stringify(scoreObject),
+  });
+};
+
```

Comment:
- [ ] I am unable to post a score, as I am getting the following error:
![image](https://user-images.githubusercontent.com/81932478/186688667-94ab28ff-3aa5-4e3d-8713-ccc4fca7964a.png)
#### Kindly fix this

Reply:
Hello @Munsa1, I have checked it again and it works fine, could you tell me what could be the problem, or could you try it again?

Reply:
Can you please record a video submitting a form It's still showing the same issue requested by the previous Code Reviewers.

Reply:
Please can we have a zoom call to show you please

https://microverse.zoom.us/j/91075584709?pwd=V0RONmZta0VlV2lrbWpFOUxnUmVUUT09